lib/matplotlib/ticker.py

```diff
@@ -1872,7 +1914,13 @@ def tick_values(self, vmin, vmax):
         return self.raise_if_exceeds(np.array(ticklocs))
 
 
-class AutoLocator(MaxNLocator):
```

Comment:
Yeah, I hear you.  The fact is `AutoLocator` is basically an alias for the "default locator" (though that's not obvious).  Anyone who actually cares which they want should use `MaxNLocator`.  A documentation problem, maybe?